package.json

```
@@ -644,23 +636,23 @@
         },
         {
           "command": "mdb.exportToRuby",
-          "when": "mdb.isPlayground == true"
+          "when": "mdb.isPlayground == true && connectedToMongoDB == true"
```

Maybe a dumb question, I’m not sure how `when` contents are being evaluated, but since this _looks_ like a JS-like language, should this be the following?

```suggestion
          "when": "mdb.isPlayground == true && mdb.connectedToMongoDB == true"
```

---

Thx! copy pasted without a prefix. This is what `when` is if you are curious: https://code.visualstudio.com/api/references/when-clause-contexts